main.go

```diff
@@ -38,18 +42,40 @@ func main() {
 		EnvVar: "APP_PORT",
 	})
 
+	annotationsEndpoint := app.String(cli.StringOpt{
+		Name:   "annotations-endpoint",
+		Value:  "http://test.api.ft.com/content/%v/annotations",
+		Desc:   "Endpoint to get annotations from UPP",
+		EnvVar: "ANNOTATIONS_ENDPOINT",
+	})
+
+	uppAPIKey := app.String(cli.StringOpt{
+		Name:   "upp-api-key",
+		Value:  "",
+		Desc:   "API key to access UPP",
+		EnvVar: "UPP_APIKEY",
+	})
+
+	apiYml := app.String(cli.StringOpt{
+		Name:   "api-yml",
+		Value:  "./api.yml",
+		Desc:   "Location of the API Swagger YML file.",
+		EnvVar: "API_YML",
+	})
+
 	log.SetLevel(log.InfoLevel)
-	log.Infof("[Startup] %v starting", *appName)
+	log.Infof("[Startup] %v is starting", *appSystemCode)
 
 	app.Action = func() {
 		log.Infof("System code: %s, App Name: %s, Port: %s", *appSystemCode, *appName, *port)
 
+		annotationsAPI := annotations.NewAnnotationsAPI(*annotationsEndpoint, *uppAPIKey)
+		annotationsHandler := annotations.NewHandler(annotationsAPI)
+		healthService := health.NewHealthService(*appSystemCode, *appName, appDescription, annotationsAPI)
 		go func() {
-			serveAdminEndpoints(*appSystemCode, *appName, *port)
+			serveEndpoints(*port, apiYml, annotationsHandler, healthService)
```

Comment:
This is taken from @izzyblues - but I'm assuming that waiting for certain os signals leads to a more graceful shutdown?